docs/source/model_doc/vit.rst

```diff
@@ -0,0 +1,89 @@
+Vision Transformer (ViT)
```

Comment:
Why did you remove the copyright? Please put i back.

Reply:
Oh sorry, I thought it was a mistake. I didn't see this in tapas.rst, so deleted it.